dashboard/v1.1/src/pages/Input/Input.tsx

```diff
@@ -53,6 +72,37 @@ const Input: FC<{}> = () => {
   const [open, setOpen] = useState(false);
   const [showResponseButton, setShowResponseButton] = useState<boolean>(false);
 
+  useEffect(() => {
+    if (screenType === 'configScreen') {
+      let parameters: { key: string; value: string }[] = [];
+      let bodyValues: { key: string; value: string }[] = [];
+      let headerValues: { key: string; value: string }[] = [];
+      params?.forEach(param => {
+        parameters.push({
+          key: param.Name,
+          value: param.Value
+        });
+      });
+      body?.forEach(val => {
+        bodyValues.push({
+          key: val.Name,
+          value: val.Value
+        });
+      });
```

Comment:
Both of these look similar. A single function with 2 times call will be better to use.